packages/stg-definitions-v2/src/constant.ts

```diff
@@ -664,6 +676,9 @@ export const REWARDS: RewardsConfig = {
             [EndpointId.BSC_V2_TESTNET]: {},
             [EndpointId.OPTSEP_V2_TESTNET]: {},
             [EndpointId.ARBSEP_V2_TESTNET]: {},
+            [EndpointId.MANTLESEP_V2_TESTNET]: {
+                /* TODO this is a pool*/
+            },
```

Comment:
I am assuming this needs a config here since we are adding a testnet with pools for the first time, but unsure of what the config should look like. Looks like we need to deploy the RewardTokenMock to Mantle Sepolia Testnet first and likely list its address here?

cc: @TRileySchwarz

Reply:
This config is correct, but the TODO isn't needed. We don't actually need to deploy the mock reward token as we won't be using the famrs. 

The line above "            // We don't deploy the token on testnet since there are no pools", is meant to say "since there are no farms"

Reply:
Ahh i see thank you for clarifying!